internal/servers/controller/handlers/credentialstores/credentialstore_service.go

```diff
@@ -34,6 +35,7 @@ const (
 	caCertsField        = "attributes.ca_cert"
 	clientCertField     = "attributes.client_certificate"
 	clientCertKeyField  = "attributes.certificate_key"
+	domain              = "credential"
```

Comment:
We don't use a constant for the domain anywhere else, is there a good reason we should keep it here, or conversely, should we add it everywhere else?

Reply:
I thought we had a constant for domain in the other handler subpackages, but maybe we were not consistent with this. Looking now at the host_sets for example, it looks like we are using the same `"host"` string over and over. I think making a constant for that string would be better imo.

Reply:
Yeah that seems reasonable. I'll do it now.